core/spec/helpers/base_helper_spec.rb

```diff
@@ -52,10 +52,10 @@
 
   # Regression test for #1436
   context "defining custom image helpers" do
-    let(:product) { mock_model(Spree::Product, :images => [], :variant_images => []) }
+    let(:product) { mock_model(Spree::Product, images: [], variant_images: []) }
     before do
       Spree::Image.class_eval do
-        attachment_definitions[:attachment][:styles].merge!({:very_strange => '1x1'})
+        attachment_definitions[:attachment][:styles].merge!({very_strange: '1x1'})
```

Comment:
Use `attachment_definitions[:attachment][:styles][:very_strange] = '1x1'` instead of `attachment_definitions[:attachment][:styles].merge!({very_strange: '1x1'})`.<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.